src/components/entries/FEEL/Feel.js

```diff
@@ -152,15 +146,8 @@ function FeelTextfield(props) {
     }
   };
 
-  const handleOnBlur = (e) => {
-    const trimmedValue = e.target.value.trim();;
-
-    // trim and commit on blur
-    onInput(trimmedValue);
-
-    if (onBlur) {
-      onBlur(e);
-    }
+  const handleOnBlur = () => {
+    onBlur?.(localValue);
```

Comment:
We pass the last known value instead of the event to recreate the `onInput` signature.